gollum.gemspec

```diff
@@ -24,15 +24,16 @@ Gem::Specification.new do |s|
   s.add_dependency 'gollum-lib', '~> 6.0'
   s.add_dependency 'kramdown', '~> 2.3'
   s.add_dependency 'kramdown-parser-gfm', '~> 1.1.0'
-  s.add_dependency 'sinatra', '~> 3.0'
-  s.add_dependency 'sinatra-contrib', '~> 3.0'
+  s.add_dependency 'rackup', '~> 2.1'
+  s.add_dependency 'sinatra', '~> 4.0'
+  s.add_dependency 'sinatra-contrib', '~> 4.0'
```

Comment:
What is it that we use `sinatra-contrib` for?

Reply:
For the `namespace` functionality: https://github.com/sinatra/sinatra/blob/main/sinatra-contrib/lib/sinatra/namespace.rb